### ISSUE TITLE
Nomenclature for concurrent Reports

### ISSUE THREAD

**Issue by ChaseWPDEV** [twgardner2/navy_psr#33]
Right now it looks like the system is set to flag concurrent reports as "AT Commands." Is this the correct nomenclature? There are several situations where AD receive concurrent reports.

**Comment by twgardner2** [comment]
Yeah, I’m sure you’re right. That’s logic that hasn’t been touched since
the very beginning of project and could probably use some TLC. Thanks for
catching that.
